rust/src/datatypes.rs

```diff
@@ -26,8 +26,10 @@ use std::mem::size_of;
 use std::slice::from_raw_parts;
 use std::str::FromStr;
 
+use serde_derive::{Deserialize, Serialize};
+use serde_json::*;
```

Comment:
nit: Is it necessary to import everything here?

Reply:
I was getting `unused import json_internal` error from last CI run but cannot reproduce that error locally. Let me remove this and trigger the run again.